package.json

```diff
@@ -12,7 +12,7 @@
     "lint": "npx eslint . && npm run types",
     "types": "npm run docs && node bin/schema2ts.js",
     "docs": "npx jsonschema2md -d src/schemas -o docs -x docs",
-    "commit": "git-cz"
+    "commit": "npm run types && git-cz"
```

Comment:
This isn't super useful because the generated docs won't be `git add`ed.

Reply:
but it would generate the docs and then the dev sees if they are outdated